### PR TITLE
refactor: ui tweak to make badge position fixed

### PR DESCRIPTION
**Pull request by ss-dimasm**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #

part of #192, minor UI tweak for `BottomBarItem`. The badge position is breaks when composed in wider components and a little typo font size usage on there

### CODE REVIEW

**Comment by kurtdoherty** [comment]
@ss-dimasm can you please add screenshots showing before and after? 🙏

**Comment by ss-dimasm** [comment]
@kurtdoherty , this is the visual; the badge position is more cornered than before

{E505805B-A8BA-4A97-ABF2-8366419F25F0}

and it will also have the flex: 1 1 0 property to automatically fill available space within a flex container

**Comment by ss-dimasm** [comment]
@kurtdoherty, hold on I surprised the total files changed is 32 and it's more than 1000 lines 😱😱, fixing

**Comment by kurtdoherty** [comment]
@ss-dimasm thanks for the details 🙏 This PR currently includes a bunch of changes that I think belong to other branches we've merged in. If you aren't already rebasing your branch onto latest, can you please do so. Ideally, this branch only includes the changes relevant to it; not historical changes that have been pulled in from main.

**Comment by ss-dimasm** [comment]
apologies @kurtdoherty, I didn't notice while rebasing

File: src/components/bottom-bar-item/bottom-bar-item.stories.tsx [hunk 1]
```diff
@@ -124,9 +127,11 @@ export const StyleButtonUsage: Story = {
         </ElButtonBottomBarItemContainer>
 
         <ElButtonBottomBarItemContainer onClick={action('handleClick')} aria-current={undefined} className="el-mx5">
-          <ElBottomBarItemIcon>{args?.icon}</ElBottomBarItemIcon>
-          <ElBottomBarItemLabel>{args?.children}</ElBottomBarItemLabel>
-          <ElBottomBarItemBadge role="status" />
+          <ElBottomBarItemContent>
+            <ElBottomBarItemIcon>{args?.icon}</ElBottomBarItemIcon>
+            <ElBottomBarItemBadge role="status" />
```
Comment (kurtdoherty) [inline src/components/bottom-bar-item/bottom-bar-item.stories.tsx:132]: **nit-pick:** I don't think `status` is an appropriate ARIA role. I'm not sure what is, but when I read [the MDN docs on the status role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role), it doesn't feel quite right 🤷

**Review by kurtdoherty — APPROVED** [review]
LGTM 👍

File: src/components/bottom-bar-item/bottom-bar-item.stories.tsx (hunk 1 above)
Comment (ss-dimasm) [reply src/components/bottom-bar-item/bottom-bar-item.stories.tsx:132]: > I don't think status is an appropriate ARIA role.

ah yes, I forgot to remove it for storybook styles usage after we've agree on this [feedback](https://github.com/reapit/elements/pull/193#discussion_r1833496895)

**Pull request merged**